context/app/static/js/pages/Dataset/hooks.ts

```diff
@@ -128,13 +131,14 @@ function getProcessedDatasetSection({
   hit: Required<SearchHit<ProcessedDatasetInfo>>;
   conf?: VitessceConf;
 }) {
-  const { pipeline, hubmap_id, files, metadata, visualization } = hit._source;
+  const { pipeline, hubmap_id, files, metadata, visualization, creation_action, contributors } = hit._source;
 
   const shouldDisplaySection = {
     summary: true,
     visualization: visualization || Boolean(conf && 'data' in conf && conf?.data),
-    files: Boolean(files),
+    files: Boolean(files?.length !== 0),
     analysis: Boolean(metadata?.dag_provenance_list),
+    attribution: creation_action !== 'Central Process' && Boolean(contributors),
```

Comment:
Let's introduce constants for `creation_action`.

- Create Dataset Activity
- Central Process
- Multi-Assay Split
- Lab Process
- Create Publication Activity
- External Process

Reply:
Good idea - I wrote up a type `CreationAction` that represents that set of strings. Is that what you had in mind, or would creating individually named constants to represent these strings/an enum be preferable?